assets/js/main.js

```diff
@@ -4,76 +4,26 @@ window.cpGroupsFilter = window.cpGroupsFilter || {};
 
 jQuery(($) => {
 
-	$(document).ready(function () {
+	$(document).click(function (e) {
```

Comment:
@jjroley Is this necessary? Can we get more specific than `document`?

Reply:
Also, `click()` is deprecated. Please use `.on( 'click', ...`

Reply:
@ml-costmo This was pre-existing code, not sure if we can get more specific but I can do some testing. Are you thinking just main wrapper of the filters?